docs/Alignment.md

```diff
@@ -20,4 +20,7 @@ aligned_features = extractor.align(
 )
 ```
 
-For now, representational alignment is only implemented for `gLocal` and the following list of models: `clip_RN50`, `clip_ViT-L/14`, `OpenCLIP_ViT-L-14_laion400m_e32`, `OpenCLIP_ViT-L-14_laion2b_s32b_b82k` `dinov2-vit-base-p14`, `dinov2-vit-large-p14`, `dino-vit-base-p16`, `dino-vit-base-p8`. However, we plan to extend both the type of representation alignment and the range of models in future versions of `thingsvision`.
+For now, representational alignment is only implemented for `gLocal` and the following list of models: `clip_RN50`, `clip_ViT-L/14`, `OpenCLIP_ViT-L-14_laion400m_e32`, `OpenCLIP_ViT-L-14_laion2b_s32b_b82k` `dinov2-vit-base-p14`, `dinov2-vit-large-p14`, `dino-vit-base-p16`, `dino-vit-base-p8`, `resnet18`, `resnet50`, `vgg16`, `alexnet`. However, we plan to extend both the type of representation alignment and the range of models in future versions of `thingsvision`. 
+
+
+Caution: For `resnet18`, `resnet50`, `vgg16`, and `alexnet` gLocal does not achieve a Pareto frontier between ML downstream task performance and human alignment. While gLocal improves alignment with human similarity judgments for these models, it deteriorates their ML downstream task performance (such as few-shot learning and out-of-distribution detection). If you are not interested in ML downstream task performance, you can safely ignore this.
```

Comment:
Nit pick: afaik the Pareto front includes all solutions that are not dominated in all objectives (alignment, downstream task) at the same time by any other solution at the same time. If that's true, aligned imagenet models would still be on the Pareto front, but just not best-of-both-worlds.

Reply:
@a1247418 I can change the wording if you want me to.

Reply:
@a1247418 I agree with you and changed the wording accordingly.